tensorflow/core/util/mkl_util.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Nit:
```suggestion
/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Reply:
Aah! had the change ready but forgot to commit :). Good catch! Made other changes too. Please take a look.